.travis.yml

```diff
@@ -48,7 +48,7 @@ install:
   - pip install nose flake8 hacking mock
 
 script:
-  - nosetests -a '!gpu' tests/chainer_tests
+  - PYTHONWARNINGS='module::DeprecationWarning' nosetests -a '!gpu' tests/chainer_tests
```

Comment:
Is there some reason why you chose `module` as the (actions)[https://docs.python.org/2.7/library/warnings.html#the-warnings-filter], especially instead of always?